pkg/apis/crd/v1alpha1/types.go

```diff
@@ -51,6 +51,15 @@ const (
 	ActionForwardedOutOfOverlay TraceflowAction = "ForwardedOutOfOverlay"
 )
 
+type TraceflowRejectAction string
+
+const (
```

Comment:
But why we need to define reject actions? The behavior is fixed right - TPCis always reset, and others are always ICMPProhibited. In my mind, we just need a new TraceflowAction called ActionRejected.

Reply:
For the "reject" implementation in Antrea, it is a "drop" action with responding specific message. Thus I use another "rejectAction" property to store this information, but I'm OK to discard this new property and use action=ActionRejected instead.
@tnqn @antoninbas Any comments?

Reply:
I think we should drop it for now until we give users the ability to configure the reject action (if we ever do that...)

Reply:
Thanks. I dropped current design and use action=ActionRejected instead.